priam/src/main/java/com/netflix/priam/backup/AbstractBackup.java

```diff
@@ -165,10 +165,11 @@ public boolean isValidBackupDir(File keyspaceDir, File columnFamilyDir, File bac
             return false;        	
         }
 
-        String columnFamilyName = columnFamilyDir.getName();
+        String dirName = columnFamilyDir.getName();
 
+        String columnFamilyName = dirName.split("-")[0];
```

Comment:
This assumes that directory for CF is a hyphen.  Even though the code is backward compatible, should the delimiter not be a configurable property?